src/components/specimen/components/contentBlock/SpecimenOverview.tsx

```diff
@@ -0,0 +1,155 @@
+/* Import Dependencies */
+import { Row, Col, Card } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector } from 'app/hooks';
+import { getSpecimen } from "redux/specimen/SpecimenSlice";
+
+/* Import Styles */
+import styles from 'components/specimen/specimen.module.scss';
+
+
+const SpecimenOverview = () => {
+    /* Base variables */
+    const specimen = useAppSelector(getSpecimen);
+
+    /* Check for Organization logo */
+    const logo = (ror: string) => {
+        try {
+            return require(`../../../../webroot/img/organizationLogo/${ror}.png`);
```

Comment:
Make sure that everything will function when the logo is not there. For some smaller organisations we might never get a logo

Reply:
Yes, that is why I use the try/catch combination, if the file does not exist, it does not break the code at the moment. Personally I would prefer to still do it a bit different because the require/import is pretty ugly due to all the ../../../, something to look at in the future when we collect more logos